src/main/java/com/marklogic/appdeployer/command/groups/DeployGroupsCommand.java

```diff
@@ -1,20 +1,33 @@
 package com.marklogic.appdeployer.command.groups;
 
+import java.io.File;
+
 import com.marklogic.appdeployer.command.AbstractResourceCommand;
 import com.marklogic.appdeployer.command.CommandContext;
 import com.marklogic.appdeployer.command.SortOrderConstants;
-import com.marklogic.mgmt.resource.ResourceManager;
 import com.marklogic.mgmt.SaveReceipt;
+import com.marklogic.mgmt.api.server.Server;
+import com.marklogic.mgmt.resource.ResourceManager;
+import com.marklogic.mgmt.resource.appservers.ServerManager;
 import com.marklogic.mgmt.resource.groups.GroupManager;
-
-import java.io.File;
+import com.marklogic.rest.util.Fragment;
 
 public class DeployGroupsCommand extends AbstractResourceCommand {
 
+	private Server adminServerTemplate;
+	private static final String ADMIN_SERVER_NAME = "Admin";
+	private static final String DEFAULT_ADMIN_SERVER_URL_REWRITER = "rewriter.xqy";
+
     public DeployGroupsCommand() {
         setExecuteSortOrder(SortOrderConstants.DEPLOY_GROUPS);
         setUndoSortOrder(SortOrderConstants.DELETE_GROUPS);
+        adminServerTemplate = new Server(null, ADMIN_SERVER_NAME);
+    	adminServerTemplate.setUrlRewriter(DEFAULT_ADMIN_SERVER_URL_REWRITER);
     }
+    
+	public void setAdminServerTemplate(Server adminServerTemplate) {
```

Comment:
Add a getter too - that way, a client can manipulate the already-instantiated object, and thus doesn't have to know all of the things that have been configured already. Also, move the getter/setter to the bottom of the file for consistency.